### DISCUSSION TITLE
Suggestions from Nextra

### DISCUSSION THREAD

**Discussion by chemlord** [imfing/hextra#426] [Ideas]
Hi Xin, 

So I know you took inspiration from Nextra to make the new and (much better in my opinion) Hextra, I was wondering if we can implement a few things from Nextra:

1.  I was wondering if in the table of contents we could have the highlight of headers and subheaders on the table of contents when you click on them like seen on Nextra

![image](https://github.com/user-attachments/assets/64017d3c-bd12-46ee-9037-ec0b9c7eef6c)

2.  I was wondering if we can get a banner that can be displayed above the navbar that can be dismissed.
![image](https://github.com/user-attachments/assets/eb30f982-995e-4480-a189-5644d8456bcf)
https://nextra.site/docs/docs-theme/theme-configuration#banner-key

3.  I was wondering if we could also have a hide sidebar button like in nextra too.

![image](https://github.com/user-attachments/assets/b131dd3a-8bfb-4b26-8dab-cb5f4983a0f2)

**Comment by imfing** [comment]
Hi, thank you for the feedback. 

The thing is that re-implementing them for a static website can be really tedious. 
I'm open to adding these features in Hextra, but there won't be any timeline around these.
